android/settings.gradle

```diff
@@ -1,12 +1,8 @@
 rootProject.name = 'GoldWallet'
-include ':react-native-config'
```

Comment:
are u sure u wanna remove all react-native-config in android ?

Reply:
yes as it autolinked and in version 0.65.1 cause crash

Reply:
it work as previously without any problems